packages/aws-cdk/lib/init-templates/v1/app/typescript/cdk.template.json

```diff
@@ -1,3 +1,7 @@
 {
-  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts"
+  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts",
+  "watch": {
+    "include": "./**/*.ts",
```

Comment:
I'd include node_modules here for completeness (also in javascript). 

I know it's implicitly added but still